epipearl/endpoints/admin.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+
+class Admin( object ):
+
+    @classmethod
```

Comment:
i like class methods too sometimes, but not sure what the value is here unless you're intending to subclass Admin. Seems like this could be a generic module with two functions: `get_admin_params` and `set_admin_params`.